www/common_lib.inc

```diff
@@ -2208,8 +2208,8 @@ function ShardKey($test_num) {
 * @param mixed $relative_url
 */
 function SendAsyncRequest($relative_url) {
-  $protocol = ((isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] == 'on') || (isset($_SERVER['HTTP_SSL']) && $_SERVER['HTTP_SSL'] == 'On')) ? 'https' : 'http';
-  $url = "$protocol://{$_SERVER['HTTP_HOST']}$relative_url";
+  // hardcode protocol and URL to force using the loopack interface
+  $url = "http://localhost$relative_url";

```

Comment:
This will only work on port 80. Won't that cause issues working with local instances in dev?